routers/web/web.go

```diff
@@ -158,8 +178,13 @@ func Routes(sessioner func(http.Handler) http.Handler) *web.Route {
 	// Removed: toolbox.Toolboxer middleware will provide debug information which seems unnecessary
 	common = append(common, context.Contexter())
 
+	group := buildAuthGroup()
+	if err := group.Init(); err != nil {
```

Comment:
Why is that call missing in `routers/api/v1/api.go`?

Reply:
Added.